1_fetch.yml

```diff
@@ -48,6 +48,11 @@ targets:
   precip_config:
     command: viz_config[I(c('precip_cols', 'precip_bin'))]
 
+  legend_styles:
+    command: viz_config[I(c('gage_flood_col','gage_norm_col','hurricane_col','storm_name'))]
+
+  hurricane_col:
+    command: viz_config[[I('hurricane_col')]]
```

Comment:
I'm not sure I agree with moving this graphical config info into 1_fetch.yml. Could you make a case for it here?

Reply:
hmm... I don't think I made that change... Are we looking at the same version of this PR ? I think this must have been merged from an earlier PR

Reply:
I can move it to `6_visuzlize.yml` though. I think it'd fit better there

Reply:
interesting...maybe i'm seeing the extra changes because you opened this PR last night. could you close this one and open a new PR from the same branch? might be cleaner.

Reply:
I think I made that change